src/main/features/claimant-response/helpers/settlementHelper.ts

```diff
@@ -0,0 +1,65 @@
+import { Settlement } from 'claims/models/settlement'
+import { PartyStatement } from 'claims/models/partyStatement'
+import { DraftClaimantResponse } from 'claimant-response/draft/draftClaimantResponse'
+import { PaymentPlan } from 'common/calculate-payment-plan/paymentPlan'
+import { Offer } from 'claims/models/offer'
+import { Moment } from 'moment'
+import { StatementType } from 'offer/form/models/statementType'
+import { MadeBy } from 'offer/form/models/madeBy'
+import { FullAdmissionResponse } from 'claims/models/response/fullAdmissionResponse'
+import { PartialAdmissionResponse } from 'claims/models/response/partialAdmissionResponse'
+import { Claim } from 'claims/models/claim'
+import { getPaymentPlan } from 'claimant-response/helpers/paymentPlanHelper'
+import { MomentFormatter } from 'utils/momentFormatter'
+import { NumberFormatter } from 'utils/numberFormatter'
+import { PaymentScheduleTypeViewFilter } from 'claimant-response/filters/payment-schedule-type-view-filter'
+
+export function getRepaymentPlanOrigin (settlement: Settlement): string {
+
+  if (!settlement) {
+    throw new Error('settlement must not be null')
+  }
+
+  const partyStatement: PartyStatement = settlement.partyStatements.slice(-2)[0]
```

Comment:
can you make this var more descriptive? i.e. you're slicing and taking the first of something, but its not really clear what you're doing